python/ray/exceptions.py

```diff
@@ -747,6 +747,18 @@ class ObjectRefStreamEndOfStreamError(RayError):
     pass
 
 
+@DeveloperAPI
+class ActorUnavailableError(RayError):
+    """Raised when the actor is temporarily unavailable but may be available later."""
+
+    def __init__(self, error_message: str, actor_id: Optional[bytes]):
+        self.error_message = error_message
+        self.actor_id = ActorID(actor_id) if actor_id is not None else None
+
+    def __str__(self):
+        return f"The actor {self.actor_id} is unavailable: {self.error_message}"
```

Comment:
```suggestion
        return f"The task {task ID} could not be completed because the actor {self.actor_id} is unavailable: {self.error_message}.\n\nThis task will be automatically retried if `max_retries` is set and there are retries remaining."
```

(or something along these lines, these aren't the right variables)